src/sections/safe.tsx

```diff
@@ -8,21 +7,117 @@ import { useAppDispatch, useAppSelector } from '../store';
 import Section from '../future-hopr-lib-components/Section';
 import { useSigner } from '../hooks';
 import { actionsAsync } from '../store/slices/safe/actionsAsync';
```

Comment:
Move this under // Stores

Reply:
i can change this in this pr but it was not part of the existing changes.